lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -259,24 +255,18 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
      *
      * @throws \Doctrine\ORM\Mapping\MappingException
      */
-    private function reverseEngineerMappingFromDatabase()
+    protected function reverseEngineerMappingFromDatabase()
```

Comment:
Please don't change visibility of the methods: if this class isn't designed for extensibility, then don't extend from it